fe/fe-core/src/main/java/org/apache/doris/nereids/glue/LogicalPlanAdapter.java

```diff
@@ -85,6 +92,6 @@ public String toDigest() {
     }
 
     public static LogicalPlanAdapter of(Plan plan) {
-        return new LogicalPlanAdapter((LogicalPlan) plan);
+        return new LogicalPlanAdapter((LogicalPlan) plan, null);
```

Comment:
does it introduce NPE? how about remove this function, or add StatementContext parameter?

Reply:
u r right, UT and regression test not be processed now